src/Ormolu/Printer/Combinators.hs

```diff
@@ -67,12 +68,15 @@ import qualified Data.Text as T
 txt :: Text -> R ()
 txt t = ensureIndent >> spit t
 
+showMe :: Outputable a => a -> Text
+showMe = T.pack . showSDocUnsafe . ppr
```

Comment:
`showMe` is not a very good name for a function. In addition you don't add any documentation. Finally we have something like this in `Oromlu.Utils`:

```haskell
-- | Pretty-print an 'GHC.Outputable' thing.

showOutputable :: GHC.Outputable o => o -> String
showOutputable = GHC.showSDocUnsafe . GHC.ppr
```